lib/lotus/commands/new/app.rb

```diff
@@ -104,7 +104,7 @@ def upcase_app_name
         end
 
         def classified_app_name
-          Utils::String.new(app_name).classify
+          Utils::String.new(app_name).classify.tr('::', '')
```

Comment:
@davydovanton Is there any way to DRY this with `Commands::Generate::App#classified_app_name`?